<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsignarOrden</class>
 <widget class="QMainWindow" name="AsignarOrden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2120</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asignar Orden</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Asignar Orden</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboOrdenes">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboMecanicos">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>180</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAsignar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Asignar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolver">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>420</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QWidget" name="imagen" native="true">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>60</y>
      <width>411</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2120</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
